creator-node/src/utils/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { recoverPersonalSignature } from 'eth-sig-util'
```

Comment:
didnt we not want to name files utils because of the dir conflict?

Reply:
We didn't want to name files outside of the `utils/` module utils because of the conflict. It's less of a problem inside. Do you have a better name? I can change it real quick